provisioner/openid_provider.go

```diff
@@ -0,0 +1,158 @@
+package provisioner
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	stsDomain = "sts.amazonaws.com"
+	// acmRootCAThumbprint is the SHA-1 sum of the root certificate in the trust chain for the certificate use to serve
+	// the open id discovery document.
+	acmRootCAThumbprint = "9e99a48a9960b14926bb7f3b02e22da2b0ab7280"
```

Comment:
How do we update this?
SHA-1 is already under attack by crypt analysis and I believe not a safe bet for long time anymore

Reply:
As much as I would like to tell AWS that I don't think they will change what they accepts as [inputs for their API](https://github.bus.zalan.do/lago/s3-governance-control/blob/53d61110705af0e14c292ed179f6f53a21333bf6/roles/source-account-prod.yaml#L125).

Reply:
Wrong link?

Reply:
Fixed